code/core/system/proxy-fh16/include/CANMessageDataStore.h

```diff
@@ -47,11 +47,12 @@ namespace proxy {
  */
 class CanMessageDataStore : public automotive::odcantools::MessageToCANDataStore {
    public:
-    CanMessageDataStore(std::shared_ptr< automotive::odcantools::CANDevice > canDevice);
+    CanMessageDataStore(std::shared_ptr< automotive::odcantools::CANDevice > canDevice, bool readOnly = false);
```

Comment:
Shouldn't the default be true to let the developer actively make the choice?

@olbender What do you think?